metrics/web/test/auth/data/model/email_domain_validation_request_data_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:metrics/auth/data/model/email_domain_validation_request_data.dart';
+import 'package:test/test.dart';
+
```

Comment:
Seems like we should add ignore for analyzer here as we use `null` for an optional argument. Could you please check that?

Reply:
I've checked with the analyzer and this file is okay. There is an `@required` annotation in the EmailDomainValidationRequestData's constructor.